test/artifact/artifact_installation_test.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "artifact_helper"
+
+class TestArtifactInstallation < Minitest::Test
+  parallelize_me!
+
+  def test_inspec_exists
+    if windows?
+      stdout, stderr, status = Open3.capture3("Get-Command inspec")
+      assert_match(/inspec.bat/, stdout)
+    else
+      stdout, stderr, status = Open3.capture3({ "PATH" => ENV["PATH"] },
+                                              "/usr/bin/which inspec")
```

Comment:
Considering this is running _in hab_ i might be able to use which on both sides.